src/Okta.Idx.Sdk.UnitTests/IdxClientLayer1Should.cs

```diff
@@ -0,0 +1,2711 @@
+            {
+                id = "aut2ihzk1gHl7ynhd1d6",
+                methodType = "email",
+            });
+
+
+            var identifyProcessedResponse = await identifyRemediationOption.ProceedAsync(remediationProceedRequest);
+
+            identifyProcessedResponse.Should().NotBeNull();
+            identifyProcessedResponse.StateHandle.Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.Version.Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.ExpiresAt.Value.Should().Be(DateTimeOffset.Parse("2020-10-21T20:58:13.000Z"));
+            identifyProcessedResponse.Intent.Should().Be("LOGIN");
+
+            identifyProcessedResponse.Remediation.GetRaw().Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.Remediation.Type.Should().Be("array");
+
+            identifyProcessedResponse.Remediation.RemediationOptions.Should().NotBeNullOrEmpty();
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Rel.Should().Contain("create-form");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Name.Should().Be("challenge-authenticator");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Href.Should().Be("https://foo.okta.com/idp/idx/challenge/answer");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Method.Should().Be("POST");
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Accepts.Should().Be("application/ion+json; okta-version=1.0.0");
+            identifyProcessedResponse.IsLoginSuccess.Should().BeFalse();
+
+            identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Form.Should().NotBeNullOrEmpty();
+            var stateHandleFormItem = identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Form.FirstOrDefault(x => x.Name == "stateHandle");
+            stateHandleFormItem.Should().NotBeNull();
+            stateHandleFormItem.Required.Should().BeTrue();
+            stateHandleFormItem.GetProperty<string>("value").Should().Be("02zLArAwt9TEyKx8rvNzIREZTf6OCvoCbRf9gH4tU1");
+            stateHandleFormItem.Visible.Should().BeFalse();
+            stateHandleFormItem.Mutable.Should().BeFalse();
+
+            var credentialsFormItem = identifyProcessedResponse.Remediation.RemediationOptions.FirstOrDefault().Form.FirstOrDefault(x => x.Name == "credentials");
+            credentialsFormItem.Should().NotBeNull();
+            credentialsFormItem.Type.Should().Be("object");
+            credentialsFormItem.Required.Should().BeTrue();
+            var credentialsFormItemValues = credentialsFormItem.Form.GetArrayProperty<FormValue>("value");
+            credentialsFormItemValues.FirstOrDefault().Name.Should().Be("passcode");
+            credentialsFormItemValues.FirstOrDefault().Label.Should().Be("Enter code");
+
```

Comment:
Looking at the content of this test, I feel like it might be useful to have a short comment describing what this test is doing and why, it isn't obvious from the name of the test.  It appears to be validating that the json was parsed correctly and values landed where they're expected, I think a comment would reinforce that understanding.